[2024-04-10T00:43:12.468+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_congress_data.bq_create_bills_Congress_partitioned_table_task manual__2024-04-10T00:36:19.141018+00:00 [queued]>
[2024-04-10T00:43:12.474+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_congress_data.bq_create_bills_Congress_partitioned_table_task manual__2024-04-10T00:36:19.141018+00:00 [queued]>
[2024-04-10T00:43:12.474+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 3
[2024-04-10T00:43:12.481+0000] {taskinstance.py:2217} INFO - Executing <Task(BigQueryInsertJobOperator): bq_create_bills_Congress_partitioned_table_task> on 2024-04-10 00:36:19.141018+00:00
[2024-04-10T00:43:12.489+0000] {standard_task_runner.py:60} INFO - Started process 3418 to run task
[2024-04-10T00:43:12.492+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'upload_congress_data', 'bq_create_bills_Congress_partitioned_table_task', 'manual__2024-04-10T00:36:19.141018+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/upload_congress_data.py', '--cfg-path', '/tmp/tmpwd6qx7ci']
[2024-04-10T00:43:12.495+0000] {standard_task_runner.py:88} INFO - Job 24: Subtask bq_create_bills_Congress_partitioned_table_task
[2024-04-10T00:43:12.541+0000] {task_command.py:423} INFO - Running <TaskInstance: upload_congress_data.bq_create_bills_Congress_partitioned_table_task manual__2024-04-10T00:36:19.141018+00:00 [running]> on host f811dc6f69ec
[2024-04-10T00:43:12.587+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_congress_data' AIRFLOW_CTX_TASK_ID='bq_create_bills_Congress_partitioned_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-10T00:36:19.141018+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-10T00:36:19.141018+00:00'
[2024-04-10T00:43:12.590+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-04-10T00:43:12.590+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-10T00:43:12.591+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-04-10T00:43:12.635+0000] {bigquery.py:2811} INFO - Executing: {'query': {'query': 'CREATE OR REPLACE TABLE Congress.bill_status         PARTITION BY latestActionDate_formatted         AS         SELECT *, DATE(billStatus.bill.latestAction.actionDate) latestActionDate_formatted         FROM Congress.bill_status_external_table;', 'useLegacySql': False}}'
[2024-04-10T00:43:12.636+0000] {bigquery.py:1613} INFO - Inserting job ***_upload_congress_data_bq_create_bills_Congress_partitioned_table_task_2024_04_10T00_36_19_141018_00_00_ac6f9218d22d53227527cda619cc8a72
[2024-04-10T00:43:14.639+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2870, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.11/site-packages/google/cloud/bigquery/job/query.py", line 1595, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.11/site-packages/google/cloud/bigquery/job/query.py", line 1584, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.11/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Error while reading table: Congress.bill_status_external_table, error message: JSON parsing error in row starting at position 0: No such field: bill. File: gs://congress_data/bills/hconres/BILLSTATUS-118hconres30.json; reason: invalid, location: gs://congress_data/bills/hconres/BILLSTATUS-118hconres30.json, message: Error while reading table: Congress.bill_status_external_table, error message: JSON parsing error in row starting at position 0: No such field: bill. File: gs://congress_data/bills/hconres/BILLSTATUS-118hconres30.json; reason: invalid, message: Error while reading table: Congress.bill_status_external_table, error message: JSON processing encountered too many errors, giving up. Rows: 1; errors: 1; max bad: 0; error percent: 0; reason: invalid, location: gs://congress_data/bills/hconres/BILLSTATUS-118hconres25.json, message: Error while reading table: Congress.bill_status_external_table, error message: JSON parsing error in row starting at position 0: No such field: bill. File: gs://congress_data/bills/hconres/BILLSTATUS-118hconres25.json; reason: invalid, message: Error while reading table: Congress.bill_status_external_table, error message: JSON processing encountered too many errors, giving up. Rows: 2; errors: 2; max bad: 0; error percent: 0

Location: US
Job ID: airflow_upload_congress_data_bq_create_bills_Congress_partitioned_table_task_2024_04_10T00_36_19_141018_00_00_ac6f9218d22d53227527cda619cc8a72

[2024-04-10T00:43:14.670+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=upload_congress_data, task_id=bq_create_bills_Congress_partitioned_table_task, execution_date=20240410T003619, start_date=20240410T004312, end_date=20240410T004314
[2024-04-10T00:43:14.683+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 24 for task bq_create_bills_Congress_partitioned_table_task (400 Error while reading table: Congress.bill_status_external_table, error message: JSON parsing error in row starting at position 0: No such field: bill. File: gs://congress_data/bills/hconres/BILLSTATUS-118hconres30.json; reason: invalid, location: gs://congress_data/bills/hconres/BILLSTATUS-118hconres30.json, message: Error while reading table: Congress.bill_status_external_table, error message: JSON parsing error in row starting at position 0: No such field: bill. File: gs://congress_data/bills/hconres/BILLSTATUS-118hconres30.json; reason: invalid, message: Error while reading table: Congress.bill_status_external_table, error message: JSON processing encountered too many errors, giving up. Rows: 1; errors: 1; max bad: 0; error percent: 0; reason: invalid, location: gs://congress_data/bills/hconres/BILLSTATUS-118hconres25.json, message: Error while reading table: Congress.bill_status_external_table, error message: JSON parsing error in row starting at position 0: No such field: bill. File: gs://congress_data/bills/hconres/BILLSTATUS-118hconres25.json; reason: invalid, message: Error while reading table: Congress.bill_status_external_table, error message: JSON processing encountered too many errors, giving up. Rows: 2; errors: 2; max bad: 0; error percent: 0

Location: US
Job ID: airflow_upload_congress_data_bq_create_bills_Congress_partitioned_table_task_2024_04_10T00_36_19_141018_00_00_ac6f9218d22d53227527cda619cc8a72
; 3418)
[2024-04-10T00:43:14.742+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-10T00:43:14.773+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
